WalletWasabi/WabiSabi/Client/AmountDecomposer.cs

```diff
@@ -314,7 +317,7 @@ private static Money FindChange(IEnumerable<Output> decomposition, HashSet<Outpu
 		{
 			foreach (var (sum, count, decomp) in Decomposer.Decompose(
 				target: (long)myInputSum,
-				tolerance: MinAllowedOutputAmount + FeeRate.GetFee(ScriptType.Taproot.EstimateOutputVsize()), // Assume script type with higher cost to be more permissive.
```

Comment:
From https://github.com/zkSNACKs/WalletWasabi/pull/10824/files#r1212760551 comes naturally that we don't have to worry about the removal of the fee calculation, however this is not a sanity check, so the additional worry here is if `MinAllowedOutputAmount` would be too big.

Let's consider that currently `MinAllowedOutputAmount` equals to the cost of creating a single output. Since the user creates 5-ish outputs a round, 1 output cost loss added to the total fees at the very worst case is entirely acceptable. And that is the worse case. Based on Sake, the median case is `worse case / 4`.

Reply:
The upcoming changes in naive and pre decompositions have the same rationale.